sdk/servicebus/azure-servicebus/tests/perf_tests/_test_base.py

```diff
@@ -5,124 +5,208 @@
 
-class _QueueTest(_ServiceTest):
-    queue_name = "perfstress-" + str(uuid.uuid4())
+    async def _preload_queue(self) -> None:
+        data=get_random_bytes(self.args.message_size)
 
-    def __init__(self, arguments):
-        super().__init__(arguments)
-        connection_string = self.get_from_env("AZURE_SERVICEBUS_CONNECTION_STRING")
-        self.async_mgmt_client = ServiceBusAdministrationClient.from_connection_string(connection_string)
+        current_queue_message_count = 0
 
-    async def global_setup(self):
-        await super().global_setup()
-        await self.async_mgmt_client.create_queue(self.queue_name, max_size_in_megabytes=MAX_QUEUE_SIZE)
+        async with ServiceBusAdministrationClient.from_connection_string(self.connection_string) as admin_client:
+            queue_properties = await admin_client.get_queue_runtime_properties(self.queue_name)
+            current_queue_message_count = queue_properties.active_message_count
+    
 
-    async def global_cleanup(self):
-        await self.async_mgmt_client.delete_queue(self.queue_name)
-        await super().global_cleanup()
+        print(f"The current queue {self.queue_name} has {current_queue_message_count} already")
 
-    async def close(self):
-        await self.async_mgmt_client.close()
-        await super().close()
+        async with self.async_servicebus_client.get_queue_sender(self.queue_name) as sender:
+            batch = await sender.create_message_batch()
 
+            for i in range(current_queue_message_count, self.args.preload):
+                try:
+                    batch.add_message(ServiceBusMessage(data))
+                except ValueError:
+                    await sender.send_messages(batch)
+                    print(f"Loaded {i} messages")
+                    batch = await sender.create_message_batch()
+                    batch.add_message(ServiceBusMessage(data))
 
-class _SendTest(_QueueTest):
-    def __init__(self, arguments):
-        super().__init__(arguments)
-        connection_string = self.get_from_env("AZURE_SERVICEBUS_CONNECTION_STRING")
-        self.async_mgmt_client = ServiceBusAdministrationClient.from_connection_string(connection_string)
-        self.sender = self.service_client.get_queue_sender(self.queue_name)
-        self.async_sender = self.async_service_client.get_queue_sender(self.queue_name)
+            if len(batch):
+                await sender.send_messages(batch)
     
-    async def close(self):
-        self.sender.close()
-        await self.async_sender.close()
+    async def global_setup(self) -> None:
+        await super().global_setup()
+        await self._preload_queue()
+    
+    async def close(self) -> None:
+        self.receiver.close()
+        await self.async_receiver.close()
         await super().close()
+    
+   
 
-
-class _ReceiveTest(_QueueTest):
-    def __init__(self, arguments):
+class _TopicReceiveTest(_ReceiveTest):
```

Comment:
might be good to rename to:
```suggestion
class _SubscriptionReceiveTest(_ReceiveTest):
```